src/cli-validator/index.js

```diff
@@ -43,6 +43,14 @@ program
   .option(
     '-e, --errors_only',
     'only print the errors, ignore the warnings'
+  )
+  .option(
+    '-f, --fix',
+    'automatically fix errors and make in-place changes to the API doc file'
+  )
+  .option(
+    '--fix_newfile',
```

Comment:
Perhaps `--fix-new-file`?

Reply:
Yeah I agree the syntax is awkward, I keep accidently putting --fix-newfile instead of --fix_newfile